### PR TITLE
UI page & demo title updates

### PR DESCRIPTION
**Pull request by SinaChavoshi**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3088)

### CODE REVIEW

File: frontend/src/pages/GettingStarted.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const PAGE_CONTENT_MD = `
 
 ## Build your own pipeline
 
-Build an end-to-end ML pipeline with TensorFlow Extended (TFX) [**Start Here!**](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb) (Alpha)
+Build an end-to-end ML pipeline with TensorFlow Extended (TFX) [**Start Here!**](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb) (Experimental)
```
Comment (rmgogogo) [inline frontend/src/pages/GettingStarted.tsx]: I prefer just remove the Alpha or Experimental

Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: Updated accordingly, I will hold off until noon, if there are no objections will push through.

**Comment by SinaChavoshi** [comment]
image

**Comment by numerology** [comment]
> Pre-configured classification pipeline

Do we need to brand TFX here?

> An example of end-to-end distributed training of XGBoost model.

I know there's a reason but we're claiming distributed training with `workers==1` Lol

**Comment by Bobgy** [comment]
Hi @SinaChavoshi, after https://github.com/kubeflow/pipelines/pull/3086 is in, you also need to keep `DEMO_PIPELINES` in `GettingStarted.tsx`  synced with backend sample pipelines.

**Comment by rmgogogo** [comment]
/lgtm
Thanks

File: backend/src/apiserver/config/sample_config.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "file": "/samples/core/xgboost_training_cm/xgboost_training_cm.py.yaml"
   },
   {
-    "name": "[Demo] TFX - Taxi Tip Prediction Model Trainer",
+    "name": "[Demo] Pre-configured classification pipeline",
     "description": "[source code](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252Fmaster%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb) [GCP Permission requirements](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/parameterized_tfx_oss#permission). Example pipeline that does classification with model analysis based on a public tax cab dataset.",
```
Comment (rmgogogo) [inline backend/src/apiserver/config/sample_config.json]: the [source code] link here is not to github. we may update it. it points to Notebook while Xgboost points to github.

Comment (SinaChavoshi) [reply backend/src/apiserver/config/sample_config.json]: Very good catch! updated accordingly.

**Comment by SinaChavoshi** [comment]
@Bobgy thank you, updated both sample name and getting started page for TFX sample to match "Pre-configured classification pipeline". 
There is one difference though, getting started page has a start building your own pipeline that links to notebook while samples only has a link to source code. Let me know if that looks off or it is acceptable.

**Comment by SinaChavoshi** [comment]
Canceling this PR in favor of #3086 if additional changes are needed will cover after bug bash.

**Pull request closed without merging**